blueocean-core-js/test/js/components/ReplayButton-spec.js

```diff
@@ -1,14 +1,14 @@
-/**
- * Created by cmeyers on 9/16/16.
- */
-
 import React from 'react';
 import { assert } from 'chai';
 import { shallow } from 'enzyme';
 
 import utils from '../../../src/js/utils';
 import { ReplayButton } from '../../../src/js/components/ReplayButton';
 
+
+jest.mock('../../../src/js/i18n/i18n');
```

Comment:
Sorry, this is confusing: previously, we had a "test-entrypoint" file that was loaded by karma initially and run once. It called an "enableMocks" method from the i18n library to turn them on. That is no longer run now, and the tests would fail without that mock call now. Now we enable the mock _per test file_, and Jest isolates the mocking per test file.